test/ast_cli_tester.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+import 'dart:convert';
+import 'package:angular_ast/angular_ast.dart';
+
+RecoveringExceptionHandler exceptionHandler = new RecoveringExceptionHandler();
```

Comment:
Is exceptionHandler going to change? If not, "final exceptionHandler" should be used.